app/src/org/commcare/engine/cases/CaseUtils.java

```diff
@@ -105,4 +115,79 @@ public static void purgeCases() {
                 removedCaseCount, removedLedgers, taken));
 
     }
+
+    public static DAG<String, int[], String> getFullCaseGraph(SqlStorage<ACase> caseStorage,
```

Comment:
Would be nice to move these functions into `commcare-core`, maybe I'll take that on as part of the Formplayer PR